distributed/core.py

```diff
@@ -1237,33 +1246,38 @@ def error_message(e, status="error"):
     tb_text = "".join(traceback.format_tb(tb))
     e = truncate_exception(e, MAX_ERROR_LEN)
     try:
-        e_serialized = protocol.pickle.dumps(e)
```

Comment:
nit: This renaming may be more accurate but imo it doesn't make a big difference. However, it increases the size of the diff which makes the changes more likely to cause conflicts and the PR more difficult to read.
Particularly with a small change like this, this makes the review experience a little harder than it should be

Reply:
the change is difference because mypy will otherwise complain 3 lines later that you're assigning a bytes value to a Serialize variable.